common/internal_model/src/main/scala/weco/catalogue/internal_model/identifiers/IdentifierType.scala

```diff
@@ -26,6 +26,11 @@ object IdentifierType extends Enum[IdentifierType] {
   def apply(id: String): IdentifierType =
     IdentifierType.withName(id)
 
+  case object Tei extends IdentifierType {
+    val id = "tei"
```

Comment:
I wonder if this is the best ID here – TEI describes an approach to manuscript cataloguing, these identifiers are IDs within that system. `tei-manuscript-number` or something?